appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/ProjectList.java

```diff
@@ -195,40 +225,9 @@ public void refresh() {
   // ProjectManagerEventListener interface that this is the
   // implementation of.
 
-  public void refresh(boolean needToSort) {
+  public void refresh() {
```

Comment:
Note that the reason this takes a boolean to determine whether to sort is because `onProjectAdded` is called both during project list load and whenever the user creates a new project. In project list loading, we can defer the sorting until the very end--otherwise what happens is people with large project lists resort the list on every new project, which results in O(n^2 log n) performance. It's not noticeable in the dev server where there may only be a couple projects, but it did impact users with a large number of projects. My concern is that this will reintroduce the performance issue for users with many projects/folders.